### PR TITLE
WF-62979-Update System Requirements of File Format UG

### PR DESCRIPTION
**Pull request by Suriya-Balamurugan**
Kindly review the below merge request and let me know your feedback.

<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://syncfusion.atlassian.net/browse/WF-62979">https://syncfusion.atlassian.net/browse/WF-62979</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
Update System Requirements of File Format UG.
</td>
</tr>
<tr>
<td>
Changes done
</td>
<td>
 

- Updated System Requirements page which is used for live publishing.
- Removed System Requirements page from repo, which is not used for live publishing.

</td>
</tr>
</table>

### CODE REVIEW

**Comment by MohanaselvamJothi** [comment]
@Suriya-Balamurugan , Please make CI success and let me know once content reviewed with team.

File: File-Formats/installation-and-upgrade/System-Requirements.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The system requirements for using our Syncfusion FileFormats platform are as fol
 
 ### Operating Systems
 
-Syncfusion File Formats platform can be installed on any of the following operating systems:
+Syncfusion File Format libraies are supported in any of the following operating systems:
```
Comment (carolinedolly) [inline File-Formats/installation-and-upgrade/System-Requirements.md]: Syncfusion File Format libraries are supported on any of the following operating systems:

File: File-Formats/installation-and-upgrade/System-Requirements.md [hunk 2]
```diff
@@ -29,19 +29,13 @@ Syncfusion File Formats platform can be installed on any of the following operat
...
-* Processor: x86 or x64
-* RAM : 1 GB (minimum), 2 GB (recommended)
-* Hard disc: up to 8.5 GB of available space may be required. However, 750 MB free space is required in boot drive even if you are installing in other drive.
+* Android
+* iOS
 
 ### Development Environment
 
-In order to develop applications with Syncfusion File Formats platform, you need to have installed anyone of the Visual Studio version.
+In order to develop applications with Syncfusion File Format libraries, you need to have installed anyone of the Visual Studio version.
```
Comment (carolinedolly) [inline File-Formats/installation-and-upgrade/System-Requirements.md:37]: In order to develop applications with the Syncfusion File Format libraries, you need to install any of the Visual Studio versions.

File: File-Formats/installation-and-upgrade/System-Requirements.md [hunk 3]
```diff
@@ -55,5 +49,7 @@ Syncfusion File format libraries supports the following .NET Framework and .NET
 * .NET Framework 4.0
 * .NET Framework 4.5
 * .NET Framework 4.6
-* .NET Standard 2.0 (Can be used in .NET Core 2.1/3.1, .NET 5.0 application)
+* .NET Standard 2.0 (Can be used in .NET Core 2.1/3.1, .NET 5.0, .NET 6.0 application)
+
+You can use our file format libraries in any type of .NET application such as WinForms, WPF, Web services, ASP.NET Web Forms, ASP.NET MVC, ASP.NET Core, Blazor, UWP, Xamarin, WinUI and .NET MAUI applications.
```
Comment (carolinedolly) [inline File-Formats/installation-and-upgrade/System-Requirements.md:54]: You can use our file format libraries in any type of .NET application, like WinForms, WPF, Web services, ASP.NET Web Forms, ASP.NET MVC, ASP.NET Core, Blazor, UWP, Xamarin, WinUI and .NET MAUI applications.

File: File-Formats/installation-and-upgrade/System-Requirements.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ The system requirements for using our Syncfusion FileFormats platform are as fol
 
 ### Operating Systems
 
-Syncfusion File Format libraies are supported in any of the following operating systems:
+Syncfusion File Format libraries are supported in any of the following operating systems:
```
Comment (carolinedolly) [inline File-Formats/installation-and-upgrade/System-Requirements.md:16]: Syncfusion File Format libraries are supported on any of the following operating systems:

**Review by carolinedolly — APPROVED** [review]

File: File-Formats/installation-and-upgrade/System-Requirements.md (hunk 1 above)
Comment (Suriya-Balamurugan) [reply File-Formats/installation-and-upgrade/System-Requirements.md]: Addressed

File: File-Formats/installation-and-upgrade/System-Requirements.md (hunk 2 above)
Comment (Suriya-Balamurugan) [reply File-Formats/installation-and-upgrade/System-Requirements.md]: Addressed

File: File-Formats/installation-and-upgrade/System-Requirements.md (hunk 3 above)
Comment (Suriya-Balamurugan) [reply File-Formats/installation-and-upgrade/System-Requirements.md]: Addressed

File: File-Formats/installation-and-upgrade/System-Requirements.md (hunk 4 above)
Comment (Suriya-Balamurugan) [reply File-Formats/installation-and-upgrade/System-Requirements.md]: Addressed

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**